core/docz-core/src/commands/serve.ts

```diff
@@ -11,12 +11,19 @@ export const serve = async (args: Arguments<any>) => {
 
   if (config.port) {
     cliArgs.push('--')
-    // Append gatsby option `port`to CLI args
+    // Append gatsby option `port` to CLI args
     // https://www.gatsbyjs.org/docs/cheat-sheet/#cheat_sheet-text
     cliArgs.push('--port')
     cliArgs.push(String(config.port))
   }
 
+  if (config.host) {
```

Comment:
I think we should do a more generic gatsby arguments handling, so users can use all of gatsby's cli arguments.

Something like:

```js
for (const arg in config) {
  cliArgs.push('--')
  cliArgs.push(`--${arg}`)
  cliArgs.push(String(arg))
}
```

This code will make sure to scale automatically when gatsby are adding new cli arguments.
Also, this would probably solve https://github.com/doczjs/docz/issues/1374.

What do you think @rakannimer ?

Reply:
config object now contains project config (menu, themeConfig...) other than gatsby's cli arguments.

![image](https://user-images.githubusercontent.com/13724426/74116041-b0468c80-4bec-11ea-9091-961030f50046.png)

If we want make it scale automatically, maybe add a `gatsbyCliConfig` to place these gatsby's arguments?

Reply:
Hey @alexandre-lelain, what you're saying makes sense to me, and I tried doing something like that a couple of commits ago but it resulted in some errors because of conflicts over variable names between docz config and gatsby. So while it looks redundant it might be better (for now) to explicitly pass down arguments.

Regarding #1374 it should be fixed in `2.3.0-alpha.11` but I forgot to update so thanks for the reminder !